clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java

```diff
@@ -66,7 +66,7 @@ public static NodeApiVersions create(Collection<ApiVersion> overrides) {
                 }
             }
             if (!exists) {
-                apiVersions.add(new ApiVersion(apiKey.id, apiKey.oldestVersion(), apiKey.latestVersion()));
+                apiVersions.add(new ApiVersion(apiKey, apiKey.oldestVersion(), apiKey.latestVersion()));
```

Comment:
Any reason we don't move the calls to `oldestVersion` and `latestVersion` into the constructor?